content/working_with/console/pages/site-management-page.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: bt_wiki
-title: Site Management Page
+title: Sites Page
```

Comment:
Optional:

You can also change file name to `sites-page.md` and add alias using old page name.

Reply:
Done